.github/workflows/terraform.yml

```diff
@@ -0,0 +1,27 @@
+name: Terraform
+on: [pull_request]
+
+jobs:
+  on-pull-request:
+    name: On Pull Request
+    runs-on: ubuntu-latest
+    steps:
+    - name: Filter to PR Opened
+      uses: actions/bin/filter@master
+      with:
+        args: "action 'opened|synchronize'"
+
+    - name: Terraform - Format
+      uses: hashicorp/terraform-github-actions/fmt@v0.3.5
```

Comment:
I would assume that latest version is 0.4.4 - https://github.com/hashicorp/terraform-github-actions/releases

Reply:
Good catch - I've updated this!